cli-toolchain/quickstart.md

```diff
@@ -585,6 +597,104 @@ As you can see in the prompt above, you can either choose to use an already exis
 
 ***Note***: You can associate more than one Lambda Function trigger to a DynamoDB table.
 
+##### As a part of the GraphQL API (types with @model annotation)
+
+You can also associated a Lambda trigger with an AppSync schema's DynamoDB backed @models. GraphQL mutations that result in DynamoDB item changes will in turn result in change records published to DynamoDB streams that can trigger a lambda function. To create such a function, start with adding a new lambda function with:
```

Comment:
We should be explicit that the GraphQL API must first be added before attaching a trigger otherwise it cannot select the appropriate table

Reply:
Done.